<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QDialog" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件/文件夹</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sOpenButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>83</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>打开...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sSenfButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>83</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="sStatusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>160</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择要发送的文件!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sCloseButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>83</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
